app/retail/templates/shared/testimonials.html

```diff
@@ -55,6 +55,12 @@ <h4 class="pt-1">Who's using Gitcoin?</h4>
     </div>
   </div>
 
+    <div class="row mt-2 pb-3 animatedParent" >
+        <div class="col d-flex flex-column align-items-center animatedParent youtube">
+            <iframe style="height:288px; width:500px; " src="https://www.youtube.com/embed/hpqHJCiqXzA" class="animated fadeInRightShort" frameborder="0" allowfullscreen></iframe>
+        </div>
+
```

Comment:
@mbeacom Does our linter catch empty lines?

Reply:
@mbeacom   This one :  "no-multiple-empty-lines": "error" 
Doc: [link](https://eslint.org/docs/rules/no-multiple-empty-lines)

If needed , I can shoot out a PR with this

Reply:
@thelostone-mc Thanks - https://github.com/gitcoinco/web/commit/ef1d21418e4f517b9b40763437be21e13d5205a7